docs/lib/posixlib.rst

```diff
@@ -1,4 +1,4 @@
-.. _posixlib:
+;.. _posixlib:
```

Comment:
It looks like a typo. With a semicolon at the beginning generation of the docs would give a warning
```
scala-native/docs/lib/libc.rst:85: WARNING: undefined label: posixlib (if the link has no caption the label must precede a section header)
```

Reply:
Thank you for the review & for finding the typo. 

Sorry, I must have missed it when I did my manual pre-submit build & review of
the generated Html.

Fixed now.   I double checked the generated html.